src/chains/ethereum/ethereum/tests/api/eth/instamine.test.ts

```diff
@@ -0,0 +1,138 @@
+import getProvider from "../../helpers/getProvider";
```

Comment:
I renamed this to `instamine.test.ts` because the previous name was based off of now-obsolete iterations on the name for the instamine modes.

I've also rearranged the describe blocks a bit to make more sense. I'll mark below where the actual new test is.